geowebcache/core/src/main/java/org/geowebcache/util/FileUtils.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.commons.logging.LogFactory;
 
 public class FileUtils {
-    private static Log log = LogFactory.getLog(org.geowebcache.util.FileUtils.class);
+    static Log log = LogFactory.getLog(org.geowebcache.util.FileUtils.class);
```

Comment:
Log is changed from Private to package-private here, but public to private above? Can we be consistent?

Also, log vs LOGGER - can we be consistent?